docs/core/config/dict.md

```diff
@@ -81,51 +81,60 @@ See [[link,auth_ldap]].
 
 ::: code-group
 ```[dovecot.conf]
-dict_legacy {
-  somedict = ldap:/path/to/dovecot-ldap-dict.conf.ext
+dict_server {
+  dict ldap {
+    driver = ldap
+    ldap_uris = ldap://{{LDAPHOST}}
+    ldap_auth_dn = uid=testadmin,cn=users,dc=dovecot,dc=net
+    ldap_auth_dn_password = testadmin
+    ldap_timeout_secs = 5
+    ldap_base = dc=dovecot,dc=net
+    ldap_starttls = no
+    ssl_client_require_valid_cert = no
+
+    dict_map priv/test/home {
+      ldap_filter = (&(homeDirectory=*)(uid=%{user}))
+      value = %{ldap:homeDirectory}
+    }
+  }
 }
 ```
 :::
 
-#### LDAP Parameters
-
-| Parameter | Required | Description |
-| --------- | -------- | ----------- |
-| `uri` | **YES**  | LDAP connection URI as expected by OpenLDAP. |
-| `bind_dn` | NO | DN or upn to use for binding. (default: none) |
-| `debug` | NO | Enable debug. `0` = off (default), `1` = on. |
-| `password` | NO | Password to use, only SIMPLE auth is supported at the moment. (default: none) |
-| `timeout` | NO | How long to wait for reply, in seconds. (default:30 seconds) |
-| `max_idle_time` | NO | Disconnect from LDAP server after connection has been idle for this many seconds. (default: never) |
-| `tls` | NO | Use TLS?<br/>`yes`: Require either ldaps or successful start TLS<br/> `try`: Send start TLS if necessary (default)<br/> `no`:  Do not send start TLS. |
+#### LDAP Settings
+
+| Parameter | Mandatory | Description |
+| --------- | --------- | ----------- |
+| `ldap_uris` | **YES**  | LDAP connection URI as expected by OpenLDAP. |
+| `ldap_auth_dn` | NO | DN or upn to use for binding. |
```

Comment:
These should be replaced by SettingsComponent and settings moved to data/settings.js. You can probably start from what Stephan added to Sieve (they shouldn't be there) in https://github.com/dovecot/documentation/pull/1100

Reply:
Oh, except they are already there. So just change to SettingsComponent.

Reply:
I also have this change in https://github.com/dovecot/documentation/pull/1107